<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recent_games</class>
 <widget class="QWidget" name="recent_games">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1570</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>741</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 700 48pt &quot;Script MT&quot;;
	background-color: rgb(0, 255, 0);

}</string>
   </property>
   <property name="text">
    <string>         Recent Games</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>190</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>matchNumber :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>530</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 2 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>290</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>opponent :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>700</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>final result :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 1 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>620</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 3 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>360</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>690</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>final result :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>matchNumber :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 2 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>opponent :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 1 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>610</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 3 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>700</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>final result :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>matchNumber :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>opponent :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>610</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 3 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 2 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>round 1 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="match_number1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>189</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="opponent_1">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>279</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="date_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>349</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round1_1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>429</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round2_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round3_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>610</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="final_result_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>690</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="date_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>349</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="final_result_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>700</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="opponent_2">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>279</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round3_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>610</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="match_number1_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>189</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round1_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>429</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round2_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>520</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="date_3">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>359</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="final_result_3">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>700</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="opponent_3">
   <property name="geometry">
    <rect>
     <x>1232</x>
     <y>289</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round3_3">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>620</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="match_number1_3">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>199</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round1_3">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>439</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="round2_3">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>530</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
